### ISSUE TITLE
Editable history

### ISSUE THREAD

**Issue by andreirtaylor** [andreirtaylor/MoneyTracker#32]
Having a history that you can edit would be ideal. Not sure if this is a realistic goal or not but it would be interesting to look into how hard it would be to implement.

**Comment by adamtheschmidt** [comment]
History of money transfers that is?

**Comment by andreirtaylor** [comment]
Yeah, something that you can click and look at the last few transfers. Then revert them if you want.
